plugins/hydra_nevergrad_sweeper/example/config.yaml

```diff
@@ -12,19 +12,19 @@ hydra:
         budget: 100
         # number of parallel workers for performing function evaluations
         num_workers: 10
-        #parametrization:
-        #  db:
-        #    - mnist
-        #    - cifar
-        #  lr:
-        #    init: 0.02
-        #    step: 2.0
-        #    log: true
-        #  dropout:
-        #    lower: 0.0
-        #    upper: 1.0
-        #  batch_size: [4, 8, 12, 16]
-
+      parametrization:
+        db:
+          - mnist
+          - cifar
+        lr:
+          init: 0.02
+          step: 2.0
+          log: true
+        dropout: Scalar(lower=0.0, upper=1.0)
+        batch_size:
+          lower: 4
+          upper: 16
+          integer: true
```

Comment:
Is this here to showcase the support for native Nevergrad types?
While this is very cool thing to support from Nevergrad, it's a bit clunkier than maybe something like:
```
dropout:
  lower: 0.0
  upper: 1.0
```
The above is a bit easier to override from the command line.

Is `Scalar(lower=0.0, upper=1.0)` actually a range for optimization (similar to command line `dropout=0.0:1.0`)?

If this is here to showcase, put a comment explaining it and making a recommendation for the preferred syntax. (In my opinion it should be the more native Hydra syntax when possible).

Reply:
It's show casing indeed, and I was in the process of adding some comments (mentioning you should avoid it whenever possible)
And yes it's exactly equivalent ;)

Still need to add some (faster) test for the config parametrization and go through the doc one more time, but this start to look good.
I'll also need to release v0.4.0 of nevergrad, that solves a couple of issues.

Reply:
Sounds good.
Since we have some followup improvements anyway: both structured configs and some decisions to make with both Ax and Nevergrad in mind , I also think it's very close to landing.